tox.ini

```diff
@@ -5,15 +5,13 @@ small_test_ee = quay.io/ansible/python-base
 mypy_tmp_dir = mypy_tmp_dir
 
 [tox]
-envlist = linters, type, py38, report, clean
+envlist = linters, type, py, report, clean
 minversion = 1.4.2
 skipsdist = True
-skip_missing_interpreters = true
```

Comment:
This is good to have in the base config and it should be explicitly disabled in CI, not here.

Reply:
I still don't like this.

Reply:
That configuration settings is permanent source of false-positive testing on CI as it will make tox use a different python version than the expected one. Is one of the most common mistakes people make while writing tox.ini. 

This repository was affected by such a bug, where we run py36 for almost an year while telling everyone we test with py38. Similar bugs happened with OpenStack projects and that this why they even went so far to be explicit about disabling this anti-feature. See https://codesearch.opendev.org/?q=skip_missing_interpreters&i=nope&literal=nope&files=&excludeFiles=&repos=

If you do not care about python version, you use `py` env. If you care, you mention it and tox should ensure that it using it, regardless if this happens locally or on CI.

There is another reason why this must be fixed here first: CI system that we can use can change, we may decide to switch from one system to another and almost for sure when doing this we will forget about small details like this. Changing in tox.ini prevents further accidents.

Reply:
I do no agree with on this but I removed that change to unblock the PR.